src/main/java/org/jenkinsci/plugins/xunit/XUnitProcessor.java

```diff
@@ -69,7 +71,12 @@ public XUnitProcessor(TestType[] types, XUnitThreshold[] thresholds, int thresho
         this.extraConfiguration = extraConfiguration;
     }
 
-    public boolean performXUnit(boolean dryRun, AbstractBuild<?, ?> build, BuildListener listener)
+    public boolean performXunit(boolean dryRun, AbstractBuild<?, ?> build, BuildListener listener)
```

Comment:
It does not seem to be used outside this plugin, why are you keeping this compatibility-bridge method and didn't do the same in `XUnitBuilder#perform`?

I'm ok on breaking backward compatibility when there is no known callers, but let's follow the same rule in all places (specially within a PR).